R/PartInv.R

```diff
@@ -1,63 +1,92 @@
 #' @importFrom stats qchisq pnorm qnorm nlminb
 #' @importFrom mnormt pmnorm
-#' @import zeallot
 NULL
 
-#' Evaluating selection accuracy based on the MCAA Framework
-#'
-#' \code{PartInv, PartInvMulti_we} evaluate partial measurement invariance using
-#' an extension of Millsap & Kwok's (2004) approach
+#' Evaluating selection accuracy for two or more groups based on the MCAA Framework
```

Comment:
Should we deprecate `PartInvMulti_we()` as well?

Reply:
Yes, created an issue for it.